index.js

```diff
@@ -26,6 +28,30 @@ app.post('/pdf', (req, res) => {
         });
 });
 
+app.post('/html2pdf', multer().any(), (req, res) => {
```

Comment:
Do you think we should keep just one endpoint `/pdf` but with different `Content-Type`  for handling different input mechanisms ?
So, for the one accepting a URL in the body, `Content-Type` will be `application/json`. This one which accepts a file can be `multipart/form-data`.
I am motivated for this due to 2 reasons:-
1. A POST endpoint in REST terms basically means that you are creating a resource. In this case, the end result is the same:- a PDF file. Hence, the only difference is how we are creating the resource at the server side, something which we can [dictate to the server using a HTTP request header](https://restfulapi.net/content-negotiation/). 
2. Papyrus need not be limited to HTML but can and should be used to generate pdfs of images, .xml files, .txt files, etc ( basically anything which is readable by human eyes). As such, we might need a way by which we can support all these input types.

Reply:
Yes, that's a great idea!
Would you like that to happen as part of this PR itself or mind creating a separate issue to clean it up?
I think it can use a bit of cleaning the structure as well, like moving the controller endpoints in a folder and have separation of services/utils etc. Not sure if its the right time or not.